spec/features/merchants/dashboard_spec.rb

```diff
@@ -281,15 +289,69 @@
         visit "/merchants/#{@pretty_plumbing.id}/dashboard" 
 
         within("##{@chair.id}") do
-          expect(find_link("#{@invoice_3.id}")[:href].should == "/merchants/#{@pretty_plumbing.id}/invoices/#{@invoice_3.id}")
+          find_link({text: "#{@invoice_3.id}", href: "/merchants/#{@pretty_plumbing.id}/invoices/#{@invoice_3.id}"}).visible?
+        end
+
+        within("##{@lamp.id}") do
+          find_link({text: "#{@invoice_4.id}", href: "/merchants/#{@pretty_plumbing.id}/invoices/#{@invoice_4.id}"}).visible?
+        end
+
+        within("##{@toilet.id}") do
+          find_link({text: "#{@invoice_5.id}", href: "/merchants/#{@pretty_plumbing.id}/invoices/#{@invoice_5.id}"}).visible?
+        end
+      end
+    end
+
+    # As a merchant
```

Comment:
We'll have to pick when we go back to refactor if we want to keep all the user stories tabbed in comments or clean them out for submission